color-picker/src/commonMain/kotlin/com/godaddy/android/colorpicker/harmony/HarmonyColorPicker.kt

```diff
@@ -32,12 +32,44 @@ import kotlin.math.atan2
 import kotlin.math.hypot
 import kotlin.math.min
 
+@Deprecated(
+    message = "Use version with programmatic updates",
+    replaceWith = ReplaceWith(
+        """HarmonyColorPicker(
+            modifier = modifier,
+            harmonyMode = harmonyMode,
+            color = color,
+            onValueChanged = { HsvColor.from(it).let(onColorChanged) }
+        )"""
+    )
+)
 @Composable
 fun HarmonyColorPicker(
     modifier: Modifier = Modifier,
     harmonyMode: ColorHarmonyMode,
     color: Color = Color.Red,
     onColorChanged: (HsvColor) -> Unit
+) {
+    HarmonyColorPicker(
+        modifier = modifier,
+        harmonyMode = harmonyMode,
+        color = color,
+        onValueChanged = { HsvColor.from(it).let(onColorChanged) }
```

Comment:
As mentioned in the other comment, this should be reverted as it'll lose specific color detail.

Reply:
@riggaroo I believe I fixed this. Please let me know if that isn't the case